### PR TITLE
DOCSP-38823: Insert docs with geospatial data

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-38823
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/java/DOCSP-38823-insert-doc-geospatial/fundamentals/crud/read-operations/geo/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/geo.txt [hunk 1]
```diff
@@ -85,6 +85,23 @@ To learn more about the shapes you can use in MongoDB, see the
 
 .. external resource
 
+Insert a Document Containing GeoJSON Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To insert a document that stores GeoJSON data, specify a document that contains a GeoJSON
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/geo.txt:91]: S: to me, "specify a document" is a little funny on its own like this. looks like you used "create a document" in the next section; i think that works better

```suggestion
To insert a document that stores GeoJSON data, create a document that contains a GeoJSON
```

File: source/fundamentals/crud/read-operations/geo.txt [hunk 2]
```diff
@@ -85,6 +85,23 @@ To learn more about the shapes you can use in MongoDB, see the
...
+value and pass the document to the ``insertOne()`` method.
+
+The following example inserts a document that includes a ``location.geo`` field,
+which stores GeoJSON data:
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/geo.txt:95]:
```suggestion
which contains GeoJSON data:
```

File: source/fundamentals/crud/read-operations/geo.txt [hunk 3]
```diff
@@ -118,6 +135,24 @@ Legacy coordinate pairs have the following structure:
...
 
+Insert a Document Containing Legacy Coordinates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To insert a document that stores a legacy coordinate pair, create a document that
+assigns a coordinate pair value to a field. Then, pass the document to the
+``insertOne()`` method.
+
+The following example inserts a document that includes a ``coordinates`` field,
+which stores geospatial data as a legacy coordinate pair:
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/geo.txt:146]:
```suggestion
which contains geospatial data as a legacy coordinate pair:
```

**Review by mongoKart — APPROVED** [review]
LGTM w/ a few suggestions

**Review by katcharov — APPROVED** [review]
LGTM!

**Pull request merged**